release-notes/2024-05/README.md

```diff
@@ -0,0 +1,212 @@
+
+# SystemLink Enterprise release 2024-05 Release Notes
+
+The 2024-05 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Navigate SystemLink Enterprise faster with the intuitively organized UI.
+- Visualize data tables in a data space to gain insights into your time-series data.
+- Analyze test results with Jupyter Notebooks.
+- Annotate steps with keywords from a data space.
+- Save custom column configurations for the product specifications grid.
+- `workorder 0.4.35`
+    - As the work orders and test plans are unlinked, the privileges are split between workorders and tesplans as follows:
+        - Create work orders -  `workorder: Create`
+        - Modify work orders -  `workorder: Update`
+        - Delete work orders -  `workorder: Delete`
+        - List and view work orders -  `workorder: Query`
+        - Create test plans -  `testplan: Create`
+        - Modify test plans -  `testplan: Update`
+        - Delete test plans -  `testplan: Delete`
+        - List and view test plans -  `testplan: Query`
+        - Access work orders web application - `workorder:AccessApplication`
+        - Access test plans web application - `testplan:AccessApplication`
```

Comment:
We don't need to list the specific underlying privileges; e.g. `work order: Create` etc. Just listing the user facing strings is enough.

Reply:
Have kept only the User Facing Strings and removed the last part of each bullet